pkg/elasticsearch/cluster.go

```diff
@@ -205,7 +206,8 @@ func (ec *esClient) DoSynchronizedFlush() (bool, error) {
 	}
 
 	if payload.Error == nil && failed != 0 {
-		payload.Error = fmt.Errorf("Failed to flush %d shards in preparation for cluster restart", failed)
+		payload.Error = errors.New("failed to flush shards in preparation for cluster restart",
```

Comment:
There is no claim to be a drop-in replacement for the standard library so the signatures of `New` don't have to match. However, I'm okay with renaming the package `kverror` to avoid a collision with the standard library.